distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -160,6 +160,10 @@ type SQL_Generator
         base_expression = self.generate_expression dialect order_descriptor.expression
         base_expression ++ collation ++ order_suffix ++ nulls_suffix
 
+    ## PRIVATE
+    default_generate_column self dialect expr:(SQL_Expression | Order_Descriptor | Query) name:Text -> SQL_Builder =
```

Comment:
```suggestion
    generate_column_for_where self dialect expr:(SQL_Expression | Order_Descriptor | Query) name:Text -> SQL_Builder =
```
(If this is in fact the generator for WHERE expressions)

Reply:
No this is just the default for dbs that don't care